.github/workflows/upgrade-python-requirements.yml

```diff
@@ -12,10 +12,11 @@ on:
         default: 'master'
 
 jobs:
-   call-upgrade-python-requirements-workflow:
+  call-upgrade-python-requirements-workflow:
     with:
        branch: ${{ github.event.inputs.branch }}
-       team_reviewers: "axim-engineering"
+       team_reviewers: "axim-aximprovements"
+       email_address: "aximimprovements@axim.org"
```

Comment:
We can keep send_success_notification to false to avoid weekly success emails from many repositories.

Reply:
Done